ui/packages/consul-ui/app/components/consul/token/list/index.hbs

```diff
@@ -30,7 +30,14 @@ as |item|>
         </dl>
     </BlockSlot>
     <BlockSlot @name="actions" as |Actions|>
-
+    {{#if (and (not-eq item.SecretID '') (not-eq item.SecretID '<hidden>'))}}
```

Comment:
Granted that I'm not familiar with consul ui code, is it possible to add a `item.hasSecret` or a similar conditional method to do such string matching?

Reply:
yea, I could do a computed func in the token model to check for a valid Secret ID.